Presentations/06-R-programming-practices.Rmd

```diff
@@ -674,32 +579,39 @@ name: functions-inception
 
 # Functions inception
 
+## A function inside a function
```

Comment:
I feel somewhat we might spend much time in the first part (locating paths, here packages, setup up, etc.). And we are getting to functions or beginning to talk about programming until here.

Reply:
that's true. I do think that all of it is necessary, though, to give a better context on a project-based workflow. I'll make sure to not focus on it too much so that we can have more time to the programming part. Also, in the introduction slides I'll mention that we'll also touch a bit on programming and workflow practices